test/common/scenarios.js

```diff
@@ -57,6 +58,25 @@ function Multisig(options) {
 		passphrase: this.account.passphrase,
 		secondPassphrase: this.account.secondPassphrase,
 	});
+
+	this.transfer = (
```

Comment:
This helper has the same function than -> https://github.com/LiskHQ/lisk/blob/7a269664f65abeb2f5fffc595cbc3f75af59d3b7/test/common/scenarios.js#L52

Reply:
It's almost the same but this one accepts a recipientId instead of using the instance property. But I think based on your other comment I could move this logic to the test so the process of what's going on each test case is more obvious. What do you thing about moving it directly to each test case by using lisk-elements directly?

Reply:
@diego-G thanks for the comments about scenarios they were indeed unneeded. Changes done!